packages/manager/apps/telecom/package.json

```diff
@@ -135,7 +135,7 @@
     "ovh-manager-webfont": "^1.0.1",
     "ovh-ng-input-password": "^1.2.5",
     "ovh-ngstrap": "^4.0.2",
-    "ovh-ui-angular": "^3.9.5",
+    "ovh-ui-angular": "3.9.0",
```

Comment:
I'm not sure this is a good idea given we're in a angualr monorepository context

Reply:
This requires testing yes :)

Reply:
Yes I understand, hence testing is required :)

Reply:
@marie-j I agree that it is not a ideal solution but I'm relaying this as a hotfix.

In fact both application "Public Cloud" and "Telecom" are impacted.

As a quick fix I prefer downgrade the version of `ovh-ui-angular` in order to be able to apply the right patch.

Why? Because version [`v3.9.1`](https://github.com/ovh-ux/ovh-ui-angular/blob/master/CHANGELOG.md#391-2019-10-10) introduce this regression.

Reply:
Seems to be fixed by [`v3.9.8`](https://github.com/ovh-ux/ovh-ui-angular/blob/master/CHANGELOG.md#398-2019-10-24). So I close this Pull request.